README.md

```diff
@@ -185,52 +187,122 @@ def train():
 HOWTOs are sample diffs showing how to change various things in your training
 code.
 
-Here are a few examples.
-
-
+Here is an example.
 
-### Polyak averaging
 
-This diff shows how to modify the MNIST example above to evaluate with
-an exponential moving average of parameters over the course of training.
+### Ensemble learning
```

Comment:
ensemble training with pmap seems a bit niche. Why do we want this to be the in the top level README?
I don't see why someone needs to see an example of how howto's work. I think it makes more sense to give a link to the howto's or a showcase of 5 HOWTO's that we use to sell the concept

Reply:
People seemed to think Polyak averaging was even more niche, I think. We just want to show one HOWTO to explain what it is, and then point to the list of HOWTOs in the docs. This seems like the first HOWTO that decently short and people might care about. But we can change this going forward.

Reply:
I agree this view is not very nice, and I would be happy to replace it with a link to a diff view of the actual HOWTO. However, we currently don't have a howto branch yet, so I suggest we leave this here and replace it once we have the actual branch (which should be in a few days). WDYT?